.scripts/electron-desktop-environment/desktop-environment-manager.ts

```diff
@@ -0,0 +1,108 @@
+import { DesktopEnvirontmentContentFactory } from './desktop-environtment-content-factory';
```

Comment:
@syns2191 typo in `environtment`, fix in next PR. Make sure you are using VS Code extension to see such typos and fix before commits!